### PR TITLE
Interim update to Risk Assessment documents

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR updates the English and German cwa-risk-assessment documents.

The current documents have become out of date and they no longer correctly describe the risk assessment calculations carried out in the currently released app version, nor do they link to published information which describes how the risk assessment is now done.

The document edits point the reader to newer information regarding app versions V1.5, V1.9 and FAQ information until such time as the documents can be completely revised.

The PR is related to "Out of date Risk Assessment document" https://github.com/corona-warn-app/cwa-documentation/issues/453 and partially mitigates the issues described there.

---
Internal Tracking ID: [EXPOSUREAPP-3548](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3548)

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Info forwarded into corresponding JIRA ticket. Thanks!

File: translations/cwa-risk-assessment.de.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Wie ermittelt die Corona-Warn-App ein erhöhtes Risiko?
 
+---
+**Nachträgliche Verbesserungen der Risikoberechnung**
+
+Verbesserungen der Riskoberechnung sind mit App-Versionen [V1.5 (Symptomerfassung)](https://www.coronawarn.app/de/blog/2020-10-19-version-1-5/) bzw. [V1.9 (Exposure Notification Framework v2)](https://www.coronawarn.app/de/blog/2020-12-16-corona-warn-app-version-1-9/) nach der Veröffentlichung dieses Dokuments eingeführt worden. Im FAQ-Abschnitt [Risiko-Ermittlung](https://www.coronawarn.app/de/faq/#risk_assessment) werden diesbezügliche Fragen beantwortet.
```
[the remark this answers is not in the document]
Comment (MikeMcC399) [reply translations/cwa-risk-assessment.de.md]: @Ein-Tim 
I made the change you suggested. Thank you!

I also set up my Atom editor to do spell-checks in .md documents. I had to add ", text.md" into the Grammars list of the spell-check package.

**Comment by MikeMcC399** [comment]
@dsarkar 
Is this going to move forward? I can't see anything contentious in the proposed content which is only referring to other already published content.

**Comment by MikeMcC399** [comment]
Closing due to lack of approval and progress.

**Pull request closed without merging**